src/core/content/util.ts

```diff
@@ -14,6 +14,10 @@ const BrowserStorage = (async () => {
 	return import('@/core/storage/browser-storage');
 })();
 
+const Options = (async () => {
```

Comment:
This is really smelly. Are you 100% sure you can't just do an `import * as Options from '@/core/storage/options'` here?
I saw you described meeting some issues before, but the message you described felt like if you were to use import type instead of a normal import, or otherwise botched the import.

Reply:
Here's a commit where I've tried to do exactly that and it's causing the error I described earlier: https://github.com/blimpage/web-scrobbler/commit/fb3ccd4 I've put screenshots of the errors I'm seeing in a comment on the commit. I don't _think_ I'm doing anything wrong there, what do you reckon?

Reply:
Ah yeah I realize now this is within content. Yeah, this is fine for now. I want a better long term solution but this will do for the time being.